torch/lib/THD/base/data_channels/DataChannelTCP.cpp

```diff
@@ -70,30 +69,26 @@ DataChannelTCP::DataChannelTCP(InitMethod::Config config, int timeout)
   , _port(0)
   , _timeout(timeout)
   , _poll_events(nullptr)
+  , _processes(config.world_size)
 {
   _rank = config.rank;
 
-  if (_rank == MASTER_RANK) { // MASTER
+  if (_rank == 0) { // MASTER
```

Comment:
And then you'll have to find all the other places + those changed in this PR. I think this is a bad idea.